genai_bench/cli/option_groups.py

```diff
@@ -428,6 +574,170 @@ def distributed_locust_options(func):
     return func
 
 
+# Storage provider authentication options
+def storage_auth_options(func):
+    """Storage provider authentication options.
+
+    Supports various storage providers:
+    - OCI Object Storage
+    - AWS S3
+    - Azure Blob Storage
+    - GCP Cloud Storage
+    - GitHub Releases/Artifacts
+    """
+    # Storage provider selection
+    func = click.option(
+        "--storage-provider",
+        type=click.Choice(
+            ["oci", "aws", "azure", "gcp", "github"], case_sensitive=False
+        ),
+        default="oci",
+        help="Storage provider for uploading results. "
+        "Defaults to OCI for backward compatibility.",
+    )(func)
+
+    # Generic storage options
+    func = click.option(
+        "--storage-bucket",
```

Comment:
If `--storage-bucket` and `--storage-prefix` is the replacement to `--prefix` and `--bucket`, can we simply remove `--bucket` and `--prefix`?